src/main/flight/mixer.h

```diff
@@ -82,6 +82,7 @@ typedef struct mixerConfig_s {
     bool yaw_motors_reversed;
     uint8_t crashflip_motor_percent;
     uint8_t crashflip_expo;
+    bool linear_mixer;
```

Comment:
Better to use `uint8_t` in a parameter group to make the size explicit.

Reply:
I was thinking of it, but saw yaw_motors_reversed also uses bool

Reply:
Yeah, that one is wrong as well.